diff --git a/node_modules/v-photoswipe/src/photoswipe.vue b/node_modules/v-photoswipe/src/photoswipe.vue
index 18cc8d9..d959c67 100644
--- a/node_modules/v-photoswipe/src/photoswipe.vue
+++ b/node_modules/v-photoswipe/src/photoswipe.vue
@@ -11,13 +11,11 @@
         <div class='pswp__top-bar'>
           <div class='pswp__counter'/>
           <button class='pswp__button pswp__button--close'
-            title='关闭 (Esc)'/>
-          <button class='pswp__button pswp__button--share'
-            title='分享'/>
+            title='Esc'/>
           <button class='pswp__button pswp__button--fs'
-            title='全屏切换'/>
+            title='Fullscreen'/>
           <button class='pswp__button pswp__button--zoom'
-            title='放大/缩小'/>
+            title='Zoom In'/>
           <div class='pswp__preloader'>
             <div class='pswp__preloader__icn'>
               <div class='pswp__preloader__cut'>
